bot/bot.py

```diff
@@ -1,12 +1,38 @@
-from tokenize import Token
 import hikari
 import os
 from dotenv import load_dotenv
 
-load_dotenv()
-intents = hikari.Intents.GUILD_VOICE_STATES | hikari.Intents.GUILD_SCHEDULED_EVENTS
-intents ^= 
+import lightbulb
+
+import firebase_admin
+from firebase_admin import credentials
+from firebase_admin import db 
+
+from config import *
+
+cred = credentials.Certificate(firebase_config)
+firebase_admin.initialize_app(cred, {'databaseURL': dbUrl_config})
+
+bot = lightbulb.BotApp(
+    token=Token,
+    default_enabled_guilds=(id)
+)
```

Comment:
after you remove the usage of `config` use `TOKEN` as the environment variable for the bot token and `GUILD_ID` instead of `id`